src/api/index.tsx

```diff
@@ -0,0 +1,48 @@
+type CallApiParams = {
+  endpoint: string;
+  method?: 'GET' | 'POST' | 'PUT' | 'PATCH' | 'DELETE';
```

Comment:
Do you think it's worth using an `enum` for this? You could probably make used of the `enum` at line 14, below.

Reply:
agreed